packages/database/src/migrations/20210714053110-FirstNationWithILIGraph-modifies-data.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const reportCode = 'AU_FLUTRACKING_First_Nation_With_ILI';
+
+const permissionGroupNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM permission_group WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
+};
+
+exports.up = async function (db) {
+  const dashboardItemId = generateId();
+  const dashboardRelationId = generateId();
+  await insertObject(db, 'report', {
+    id: generateId(),
+    code: reportCode,
+    config: {
+      fetch: {
+        aggregations: ['RAW'],
+        dataElements: ['FluTracker_LGA_Percent_First_Nations_ILI'],
+      },
+      transform: [
+        'convertPeriodToWeek',
+        {
+          transform: 'select',
+          "'name'": '($row.period).substring(4, 7)',
+          '($row.period).substring(0, 4)': '$row.value',
+        },
+        {
+          transform: 'aggregate',
+          name: 'group',
+          '...': 'last',
+        },
+        {
+          transform: 'select',
+          "'timestamp'": "periodToTimestamp(('2020').concat($row.name))",
+          '...': '*',
+        },
+      ],
+    },
+    permission_group_id: await permissionGroupNameToId(db, 'Public'),
+  });
+
+  await insertObject(db, 'dashboard_item', {
+    id: dashboardItemId,
+    code: 'AU_FLUTRACKING_First_Nation_With_ILI',
```

Comment:
Every time it comes to my mind that if we can set a report_code as null in `dashboard_item` and use its own code as `report_code`, because they are mostly the same.